### DISCUSSION TITLE
@slidy/svelte - Lazyload images…🤔

### DISCUSSION THREAD

**Discussion by Valexr** [Valexr/Slidy#52] [Ideas]
Add intersectionObsever()...

**Comment by EricRovell** [comment]
Maybe we should use native `loading="lazy"` [attribute](https://caniuse.com/?search=lazy)?

**Comment by Valexr** [comment]
Maybe… but I'm not sure he'll work correctly in a Safari…🤔
& I’m not sure we needed it or not 😁template have slots & this behaviour can be make outside of component - something like in [demosite](https://github.com/Valexr/Slidy/blob/733c46eed39b6834de00a7142921b3a561cd3eec/www/src/App.svelte#L40)
